iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/MergedBus.java

```diff
@@ -87,6 +87,11 @@ public void invalidate() {
         buses.clear();
     }
 
+    @Override
+    public NetworkImpl getNetwork() {
+        return (NetworkImpl) getVoltageLevel().getSubstation().getNetwork();
```

Comment:
In AbstractBus, you use voltageLevel.getNetwork(). Do the same thing here?

Reply:
My mistake, i will change that